open-company/open-company-web
# Sentry improvements

Card: https://github.com/cljsjs/packages/pull/2043

### Review after https://github.com/cljsjs/packages/pull/2043

To test:
- load the app
- open console
- make sure you have all levels selected (Error, Warning, Info and Debug/Verbose)
- refresh the app
- filter on "Sentry"
- check you see the logs about setting up the DSN and setting your user info

To test only locally:
- repeat the steps above
- in the console enter `oc.web.lib.sentry.capture_message_BANG_("Test message")`
- check you see the message on Sentry in the right project
- check it has your user's data and the following tags: `isMobile` and `hasJWT`
- in the console enter `oc.web.lib.sentry.test_sentry()`
- check you see the error on Sentry
- check it has your user's data and the following tags: `isMobile` and `hasJWT`
